Create Intro and move 'contribution.md'  from '/docs' to '/'

jinsunee opened the pull request:
<img width="1440" alt="스크린샷 2019-10-26 오후 8 47 28" src="https://user-images.githubusercontent.com/31176502/67623928-fc500a00-f865-11e9-9efc-930f8ed9a301.png">


close #196

## Changes
- Create Intro for docs.
- Change the location of `contribution.md` from `/docs/contribution.md` to `/contribution.md`.

## How Verified
#### (Automated Tests)
- N/A

#### (Manual Tests)
- N/A

### Notes
- N/A

### References
- N/A

jinsunee committed:
Create Intro and move 'contribution.md' #196

JeffGuKang reviewed (changes requested):
Contrubution.md => contribution.md

JeffGuKang reviewed (changes requested):
Contrubution.md => contribution.md

sonypark commented on README.md:
```diff
@@ -9,3 +9,24 @@ permalink: /
+
+---------
+
+React Native, unveiled on Facebook, is a great framework for creating mobile applications iOS and Android at the same time using React.
```
I think `unveiled on Facebook` is a little awkward.
How about this sentence?

`React Native created by Facebook is a great framework for building mobile applications for iOS and Android using React.`

sonypark commented on README.md:
```diff
@@ -9,3 +9,24 @@ permalink: /
+
+React Native, unveiled on Facebook, is a great framework for creating mobile applications iOS and Android at the same time using React.
+
+This tutorial covers the essential concepts of React Native, 'Basic', 'Debugging', 'React Navigation' and 'State Management', and each chapter is organized with an example so that beginners can easily follow easily.
```
`each chapter consists of easy-to-follow examples for beginners`

This one would be more natural flow.

sonypark commented on README.md:
```diff
@@ -9,3 +9,24 @@ permalink: /
+You can say, "I know the real thing, but I'm not sure." Or, "I don't know the real thing at all."
+Through this tutorial that we focus on the basics of real native, solve some vague questions like "What is Real Native?" and "How can you develop it with iOS and Android apps with Javascript?"
+
+This document is all published as an open source to respond to the fast-changing React Native framework. If you find issues such as bugs, misspellings, and changes, please leave [Issue](https://github.com/JeffGuKang/react-native-tutorial/issues) and [Contribution](./Contribution.html).
```
`This document is all published as an open source to respond to the fast evolving React Native framework.
We are grateful to the community for contributing bugfixes and improvements.  You can take part in improving React Native Tutorial. Please read our [Contributing Guide](./Contribution.html) and leave [Issue](https://github.com/JeffGuKang/react-native-tutorial/issues) for this project.`

sonypark commented on README.md:
```diff
@@ -9,3 +9,24 @@ permalink: /
+This tutorial covers the essential concepts of React Native, 'Basic', 'Debugging', 'React Navigation' and 'State Management', and each chapter is organized with an example so that beginners can easily follow easily.
+
+You can say, "I know the real thing, but I'm not sure." Or, "I don't know the real thing at all."
+Through this tutorial that we focus on the basics of real native, solve some vague questions like "What is Real Native?" and "How can you develop it with iOS and Android apps with Javascript?"
```
`If you are new to React Native, that's fine.
This is a beginner-friendly tutorial so you can understand how React Native works and how it can build iOS and Android apps using Javascript.`

sonypark reviewed (commented):
I commented for English sentences. Please refer to the comments :)

jinsunee committed:
Fix Intro English Version and Change name 'Contribution.md' to lowercase #196

jinsunee committed:
Fix Intro English Version #196

JeffGuKang commented:
Mac's default hard drive format may not recognize letter case of git. 

https://stackoverflow.com/questions/6899582/i-change-the-capitalization-of-a-directory-and-git-doesnt-seem-to-pick-up-on-it

Try this cli.

```
git mv Contribution.md contribution.md
```

jinsunee committed:
Change file name and Modify intro korean version. #196

Change file name 'Contribution.md' and Modify some grammer of 'intro' korean version.

jinsunee committed:
Merge branch '196-create-intro-and-move-contribution' of https://github.com/ilikeu7246/react-native-tutorial into 196-create-intro-and-move-contribution
